tests/system/HTTP/Files/FileCollectionTest.php

```diff
@@ -38,11 +38,12 @@ public function testAllReturnsValidSingleFile()
     {
         $_FILES = [
             'userfile' => [
-                'name'     => 'someFile.txt',
-                'type'     => 'text/plain',
-                'size'     => '124',
-                'tmp_name' => '/tmp/myTempFile.txt',
-                'error'    => 0,
+                'name'      => 'someFile.txt',
+                'type'      => 'text/plain',
+                'size'      => '124',
+                'tmp_name'  => '/tmp/myTempFile.txt',
+                'full_path' => 'tmp/myTempFile.txt',
```

Comment:
This full path is wrong because `name` is `someFile.txt`. Please make it more realistic.

Reply:
Yes. Good point.